nectar/downloaders/threaded.py

```diff
@@ -160,6 +160,7 @@ def requests_kwargs_from_nectar_config(config):
                                          host=parsed_url.host, port=config.proxy_port)
             requests_kwargs['proxies'] = {'http': parsed_url.url, 'https': parsed_url.url}
 
+        requests_kwargs['stream'] = True # required for reading the download in chunks
```

Comment:
@jortel Damn.  I thought it was too easy!  Any suggestions as to where this should be done?